modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,45 @@
+# Import needed libraries
+from PIL import Image  # image libraries and operations
+import os, sys  # directory libraries
```

Comment:
Again, as I have pointed out in a previous review, fix this line. This line is **literally** and by literally I mean **verbatim** the example [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#Imports_formatting) uses as a _bad_ example.